contracts/controller/src/borrows_supplies.rs

```diff
@@ -170,11 +170,8 @@ impl Contract {
         let _existing_borrows = self.get_entity_by_token(Borrow, account.clone(), token_address.clone());
```

Comment:
@dariacherniak please remove this and keep only check for action paused and health_factor